### PR TITLE
RemoveQingCloud and more

### PR DESCRIPTION
**Pull request by DreamOfIce**
- Remove QingCloud
- Move serverless cloud to CI and CD
- Supplement the major cloud providers section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,15 +113,15 @@ Table of Contents
...
     * [Static Web Apps](https://azure.microsoft.com/pricing/details/app-service/static/) — Build, deploy and host static apps and serverless functions, with free SSL, Authentication/Authorization and custom domains
-    * [Storage](https://azure.microsoft.com/services/storage/) - 5GB LRS File or Blob storage
+    * [Storage](https://azure.microsoft.com/services/storage/) - 5GB LRS File or Blob storage (12mo)
     * [Cognitive Services](https://azure.microsoft.com/services/cognitive-services/) - AI/ML APIs (Computer Vision, Translator, Face detection, Bots...) with free tier including limited transactions
     * [Cognitive Search](https://azure.microsoft.com/services/search/#features) - AI-based search and indexation service, free for 10,000 documents
     * [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) - Managed Kubernetes service, free cluster management
     * [Event Grid](https://azure.microsoft.com/services/event-grid/) - 100K ops/month
-    * Full, detailed list - [https://azure.microsoft.com/free/](https://azure.microsoft.com/free/)
+    * Full, detailed list - https://azure.microsoft.com/free/
```
Comment (ripienaar) [inline README.md:124]: Why remove this link?

File: README.md [hunk 2]
```diff
@@ -486,6 +485,7 @@ Table of Contents
   * [Octopus Deploy](https://octopus.com) - Automated deployment and release-management. Free for <= 10 deployment targets.
   * [scalr.com](https://scalr.com/) - Remote state & operations backend for Terraform with full CLI support, integration with OPA and a hierarchical configuration model. Free up to 5 users.
   * [semaphoreci.com](https://semaphoreci.com/) — Free for Open Source, 100 private builds per month
+  * [Serverless Cloud](https://www.serverless.com/cloud) - Serverless Cloud lets you build Serverless APIs, DBs and Storage by using infrastrucure _from_ code approach(no yaml, no infrastructure configuration). The product is currently under public preview.
```
Comment (ripienaar) [inline README.md:488]: I dont think this is a CI/CD service, they have a small aspect of that but predominantly its a PaaS

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DreamOfIce) [reply README.md:124]: It is automatically converted to a < link > tag and does not need to be declared separately

Comment (DreamOfIce) [reply README.md:124]: Just like the gcp&aws below

File: README.md (hunk 2 above)
Comment (ripienaar) [reply README.md:488]: Remaining point here

Comment (DreamOfIce) [reply README.md:488]: Sorry,I didn't find this feature when I first experienced it. Now I found it

Comment (DreamOfIce) [reply README.md]: I recover it!

**Comment by ripienaar** [comment]
thank you

**Pull request merged**